getting-started/installation-docker.md

```diff
@@ -92,10 +92,21 @@ CREATE EXTENSION postgis;
 ```
 For more instructions on using PostGIS, [see our tutorial][tutorial-postgis].
 
+## Prebuilt with Prometheus adapter
+
+We have a Prometheus adapter and extension that, when used together, allow you to
```

Comment:
Think it would be nice to provide links for the both repos here

Reply:
Great, added in some additional comments here. New commit incoming.